### ISSUE TITLE
Failed to find all ICU components (missing: ICU_LIBRARY) Error

### ISSUE THREAD

**Issue by kgpai** [facebookincubator/velox#4900]
### Problem description

Getting this error while building Velox: 
```
 Setting CMAKE_CXX_FLAGS=-mavx2 -mfma -mavx -mf16c -mlzcnt -std=c++17 -mbmi2
FINAL CMAKE_CXX_FLAGS= -mavx2 -mfma -mavx -mf16c -mlzcnt -std=c++17 -mbmi2 -D USE_VELOX_COMMON_BASE -D HAS_UNCAUGHT_EXCEPTIONS -Wall -Wextra -Wno-unused        -Wno-unused-parameter        -Wno-sign-compare        -Wno-ignored-qualifiers        -Wno-implicit-fallthrough          -Wno-empty-body          -Wno-class-memaccess          -Wno-comment          -Wno-int-in-bool-context          -Wno-redundant-move          -Wno-array-bounds          -Wno-maybe-uninitialized          -Wno-unused-result          -Wno-format-overflow          -Wno-strict-aliasing          -Wno-type-limits          -Wno-stringop-overflow          -Wno-return-type
-- Setting ICU source to SYSTEM
CMake Error at /usr/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  Failed to find all ICU components (missing: ICU_LIBRARY) (found version
  "60.3")
```

### System information

https://app.circleci.com/pipelines/github/facebookincubator/velox/24757/workflows/aef44934-5f66-4999-acb2-02b5a00a531e/jobs/157181

### CMake log

```bash
https://app.circleci.com/pipelines/github/facebookincubator/velox/24757/workflows/aef44934-5f66-4999-acb2-02b5a00a531e/jobs/157181
```

**Comment by assignUser** [comment]
Apparently icu is not correctly installed in the image, the header files are missing but the libraries are there... very weird. Looking into what caused this now.

**Comment by assignUser** [comment]
fixed by #4908
